der/src/asn1/set_of.rs

```diff
@@ -214,6 +214,19 @@ where
         }
     }
 
+    /// Create a new [`SetOfVec`] from the given iterator.
+    ///
+    /// Note: this is an inherent method instead of an impl of the
+    /// [`FromIterator`] trait in order to be fallible.
+    pub fn from_iter<I>(iter: I) -> Result<Self>
+    where
+        I: IntoIterator<Item = T>,
+    {
+        let mut ret = Self::new();
```

Comment:
This could use `Vec::with_capacity` if it were constrained to an `ExactSizeIterator`

Reply:
Went ahead and did that in 8b55b41